examples/User Creation & Provisioning/README.md

```diff
@@ -35,18 +35,27 @@ This PowerShell script uses the Box CLI to build and create an onboarding folder
 
 ## Configure the script
 ### Update the list of employee accounts for creation
+
+The header row should look like the below:
+
+firstName,lastName,email,username
+
+NOTE 1 - EMAILS MUST BE UNIQUE ACROSS ALL OF BOX - CANNOT CREATE EMAILS USED PREVIOUSLY
+
+NOTE 2 - USERNAME MUST BE UNIQUE ACROSS YOUR BOX INSTANCE. - THIS IS USED FOR THE PERSONAL FOLDER NAME
```

Comment:
So I thought about this a bit.... and I wanted to give the user the right to pick the username... since every company already makes a unique user name for an employee with their own standard... not everyone user first letter + last name. Also... we can't just use first letter + last name automatically because that can cause errors (if there are two robert smiths at a company for example...)

I need to fix it then where it requires it to exist. That's a bug.